cfgov/legacy/static/knowledgebase/less/es-ask-base.less

```diff
@@ -625,6 +625,54 @@ legend {
 }
 
 /* ==========================================================================
+   Autocomplete
+   ========================================================================== */
+@pacific:      #0072ce;
+@gray-20:      #d2d3d5;
+@gray-5:       #f7f8f9;
+@black:        #101820; 
+@block__bg:    @gray-5;
+
+.m-autocomplete {
+    position: relative;
+}
+
+.m-autocomplete_results {
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 200px;
+    border: 1px solid #d0d0ce;
+    padding-left:  0;
+    overflow-y: scroll;
+    position: absolute;
+    z-index: 1000;
+    background-color: white;
```

Comment:
Yeah, Spanish Ask is going to keep using its current Wordpress layout for now, so I've generally just been following the approach in the legacy less files when I need to add new styles there. Hopefully it will transition over to CF in the not too distant future...